File: 2 Revalida en C++/5 Funciones/11.cpp
///funcion para resolver un acertijo
#include <bits/stdc++.h>
using namespace std;
int sociendad(int n1,int n2)
{
    int x,y,z;
    x=n1*2;
	x=x+5;
	y=x*50;
	y=y+n2;
	z=y+365;
	z=z-615;
	return z;
}
int main()
{
    int n1,n2;
    cout<<"ingrese n1:";
    cin>>n1;
    cout<<"ingrese n2:";
    cin>>n2;
    cout<<"El juego de la sociedad es= "<<sociendad(n1,n2);
    return 0;
}

File: 1 Revalida en python/5 Funciones/11.py
#funcion para resolver un acertijo
def sociedad(n1,n2):
	x=n1*2
	x=x+5
	y=x*50
	y=y+n2
	z=y+365
	z=z-615
	return z
n1=int(input("ingrese un numero"))
n2=int(input("ingrese un numero"))
print sociedad(n1,n2)